atmo/templates/atmo/detail-spark-job.html

```diff
@@ -0,0 +1,43 @@
+{% extends "atmo/base.html" %}
+{% block content %}
```

Comment:
Mind providing a `head_title` block as well? it'll be used in the `<title>` element